### PR TITLE
viu: add page

### PR DESCRIPTION
**Pull request by GiorgiBeriashvili**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
1.3.0

### CODE REVIEW

**Review by CleanMachine1 — DISMISSED** [review]

File: pages/common/viu.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# viu
+
+> A small command-line application to view images from the terminal.
+> More information: <https://github.com/atanunq/viu>.
+
+- Render an image or GIF:
```
Comment (marchersimon) [inline pages/common/viu.md:6]: I think that's an important detail to mention:
```suggestion
- Render an image or animated GIF:
```

File: pages/common/viu.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`viu {{path/to/file}}`
+
+- Render an image or GIF from the internet using `curl`:
+
+`curl -s {{https://example.com/image.png}} | viu -`
+
+- Render a transparent image with a transparent background:
```
Comment (marchersimon) [inline pages/common/viu.md:14]: I think one time is enough
```suggestion
- Render an image with a transparent background:
```

File: pages/common/viu.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`viu -t {{path/to/file}}`
+
+- Render an image with a specific width and height in pixels:
+
+`viu -w {{width}} -h {{height}} {{path/to/file}}`
+
+- Render PNG or GIF files with their original names:
+
+`viu -n {{*.{png,gif}}}`
```
Comment (marchersimon) [inline pages/common/viu.md:24]: The `*.png` is a feature of the shell, not of `viu`, so we should focus on the `-n` flag here.
```suggestion
- Render an image or GIF and display its file name:

`viu -n {{path/to/file}}`
```

**Review by marchersimon — APPROVED** [review]
Thanks @GiorgiBeriashvili

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@CleanMachine1 could you review?

**Pull request merged**